discover/portal/articles/210-setting-up/01-system-settings/03-server-administration/01-server-admin-resources.markdown

```diff
@@ -57,3 +57,7 @@ Site.
 portlet preferences become orphaned in the @product@ database. 
 
 ![Figure 1: The Resources tab of Server Administration lets you execute several server maintenance tasks.](../../../../images/server-admin-resources.png)
+
```

Comment:
@jhinkey I'm not entirely sure I understand this. Can you clarify, maybe with an example?

Reply:
Does this make sense? 

> Caching occurs at multiple levels. Sometimes higher
> caching layers aren't aware of lower caching layers. Always clear the cache at
> the lowest (most granular) layer possible, even if you've already cleared a
> higher level cache.

Reply:
Hey Russ,

I don't have an example of cache layers that you'd think should affect each other but don't. But your text for the footnote is spot on and an improvement over mine. So please go ahead and use it. :-)